projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -88,12 +75,13 @@ const ProductCard = inprops => {
 		onMouseLeave,
 	} = props;
 
+	const { ownedProducts } = getMyJetpackWindowInitialState( 'lifecycleStats' );
```

Comment:
Oh! Wow, TIL you can pass a key into `getMyJetpackInitialState( key )`!  I didn't know that previously. Thanks! 😊

Reply:
Oh yes! @robertsreberski had that idea when we were updating the state in My Jetpack 😃